report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,23 +1,66 @@
 
-\subsection{Problem Formulation}
-% repeats intro
-% Predicting major oxide compositions from \gls{libs} data presents significant computational challenges, including high dimensionality, non-linearity, multicollinearity, and the phenomenon known as matrix effects~\cite{andersonImprovedAccuracyQuantitative2017}.
-% Furthermore, the high cost of data collection often results in small datasets, complicating the task of building accurate and robust models.
+\gls{libs} is particularly suitable for the Martian environment because of its ability to perform rapid chemical analyzes remotely, creating a plasma that can be spectrally analyzed to determine the elemental composition of the vaporized material. This capability is crucial because it allows scientists to quickly and efficiently assess the geochemistry of multiple sites without physically moving the rover, thus conservatively managing the rover's limited energy and resources. The mission's focus has been on assessing past habitability, and the data gathered by \gls{chemcam} has been instrumental in identifying environments that could have supported life\cite{chemcamNasaWebsite,curiosityNasaWebsite}.
 
-Our problem definition builds upon the definitions provided in \citet{p9_paper}.
+The task of quantifying the oxides in Martian rock and soil samples begins with the \gls{libs} spectral data collected by Curiosity. This data comprises high-dimensional spectra with thousands of potential features, each corresponding to a specific element's emission lines. The computational challenge lies in accurately interpreting these complex data sets to deduce the concentrations of various elements, especially major oxides like iron, magnesium, and silicon, which are crucial for understanding Martian geology.
+Computational models take these spectral readings as input and process them through several stages of analysis. Initially, the data undergoes preprocessing to correct for any instrumental effects and to calibrate the raw spectra. This step ensures that the readings are accurate and can be reliably used for quantitative analysis. 
+The cleaned data is fed into machine learning models, which have been trained to correlate specific spectral signatures with known concentrations of elements, based on a database of Earth-based and synthetic Martian analogs.
+The output of this process is a detailed quantitative analysis of the soil or rock sample's chemical composition, expressed in weight percentages of each detected oxide\cite{wiensPreflightCalibrationInitial2013, cleggRecalibrationMarsScience2017}.
```

Comment:
I don't know if it's just me, but these 3 lines feel really similar to what we've already said earlier. They repeat a lot of concepts we've already introduced. I feel like this could either be cut entirely or rephrased to much more concisely reiteate the main points.